### PR TITLE
Add doc for `-XX:[+|-]EnsureHashed` java option

### PR DESCRIPTION
**Pull request by EricYangIBM**
Closes: #826

Signed-off-by: Eric Yang <eric.yang@ibm.com>

### CODE REVIEW

File: docs/xxensurehashed.md
Comment (hangshao0) [inline docs/xxensurehashed.md:39]: Better to add some examples if multiple `-XX:+/-EnsureHashed` are used and what it means.

**Comment by hangshao0** [comment]
You also need to add this into 0.29.0 release notes (`version0.29.md`). I don't see there is a `version0.29.md` now, so you may want to create it from `version0.28.md`

File: docs/version0.29.md
Comment (hangshao0) [inline docs/version0.29.md:38]: See https://github.com/eclipse-openj9/openj9/milestone/29. Release 0.29.0 is Java 8, 11 and 17.

**Review by hangshao0 — APPROVED** [review]
Looks good to me.

**Comment by tajila** [comment]
@doveye Please review these changes

**Comment by pshipton** [comment]
jenkins doc stage

**Comment by pshipton** [comment]
See the changes at https://eclipse-openj9.github.io/openj9-docs-staging/827

**Review by pshipton — APPROVED** [review]

**Comment by EricYangIBM** [comment]
Fixed grammatical errors:
```
The `-XX:+EnsureHashed:<classes>` option specifies the classes and the `-XX:-EnsureHashed:<classes>` option unspecifies classes that were previously specified. These options are parsed left to right.
```
->
```
The `-XX:+EnsureHashed:<classes>` option specifies the classes, and the `-XX:-EnsureHashed:<classes>` unspecifies classes that were previously specified. These options are parsed left to right.
```

**Comment by pshipton** [comment]
@tajila note we can merge this without waiting for Esther's approval, as it may not be a priority right now for the docs team to look at OpenJ9 updates.

**Comment by tajila** [comment]
> @tajila note we can merge this without waiting for Esther's approval, as it may not be a priority right now for the docs team to look at OpenJ9 updates.

Sounds good. I propose waiting until its merged in 0.30 before merging this

**Review by tajila — APPROVED** [review]

**Pull request merged**
